src/artemis/__main__.py

```diff
@@ -148,6 +149,10 @@ def put(self, experiment: str, action: Actions):
                 status_and_message = StatusAndMessage(Status.FAILED, repr(e))
             except PlanNotFound as e:
                 status_and_message = StatusAndMessage(Status.FAILED, repr(e))
+            except ValidationError as e:
+                status_and_message = StatusAndMessage(Status.FAILED, repr(e))
+                artemis.log.LOGGER.error("Invalid json parameters")
+
```

Comment:
yeah so like I said, I think it would be better to have an `except Exception ...` at the end here so that we never have the flask app handling exceptions. Otherwise, the next time there's a differnt error that's not PlanNotFound or ValidationError we will have the same issue again. Also, I would like it if we include the exception stacktrace in the log message

Reply:
OK, I think this has now been done